app/scripts/offer-transform.js

```diff
@@ -42,35 +42,75 @@ var offerTransform = (function(_, commonTransforms) {
         }
         */
         var person = {};
+        person._id = _.get(record, 'itemOffered.uri');
+        person._type = "provider";
         person.name = _.get(record, 'itemOffered.name', 'Name N/A');
         person.ethnicities = _.get(record, 'itemOffered.ethnicity');
         person.height = _.get(record, 'itemOffered.height');
         person.weight = _.get(record, 'itemOffered.weight');
         person.ages = _.get(record, 'itemOffered.age');
+
+        title = (person.name != 'Name N/A')? person.name : "";
```

Comment:
This file has numerous variables defined on the global scope instead of the local scope (using "var"), including:  title (line 53), sep (line 54), prices (line 70), webpagePhonesLen (line 142), webpageEmailsLen (line 143).